### DISCUSSION TITLE
follw: The App %E2%80%93 Luozx%E2%80%99s Digtal Garden

### DISCUSSION THREAD

**Discussion by giscus** [0ptimista/Garden#1] [Announcements]
# follw: The App %E2%80%93 Luozx%E2%80%99s Digtal Garden

Follow, 不是一个纯粹的RSS阅读器

https://garden.luozx.org/posts/follow-the-app/follow-the-app.html

**Comment by yimity** [comment]
2024-10-15 的已经被使用了。求一个新的。谢谢啦。

**Reply by 0ptimista** [reply]
抱歉，现在我的token数量还不够生成新的邀请码，还需要一段时间，到时候会更新上去。
